### DISCUSSION TITLE
Get League of Legends client in other language?

### DISCUSSION THREAD

**Discussion by blueberryfigapricotrye** [fraxiinus/ReplayBook#147] [Feature Request]
Hi,

how to get League of Legends clients in other language? So far I found older League of Legends Clients version in EnglishUS (en_US). But I am looking for older League of Legends clients version with French (fr_FR).

Can ReplayBook convert an English (en_US) League of Legends client version to a French (fr_FR) League of Legends client version?

**Comment by fraxiinus** [comment]
You still should be able to play replays from all regions on a en_US copy.

I don't think it's possible to convert one language version to another without needing requiring additional files. The best bet as always is to start keeping your own backups.

**Reply by blueberryfigapricotrye** [reply]
Exactly, I can play all replays in en_US, but I want to select another language. This site for example https://replays.xyz/old-clients has only League of Legends clients with en_US.

> The best bet as always is to start keeping your own backups.

The problem is I don't have backups of the older League of Legends client version (fr_FR).

Is there a possibility to get/download another language afterwards for the respective client version?

**Reply by fraxiinus** [reply]
None that I know of.

**Reply by blueberryfigapricotrye** [reply]
> None that I know of.

If you know a developer or a friend who works at Riot Games it would be very helpful if they could make all the older League of Legends clients available for download in all languages.

**Reply by fraxiinus** [reply]
I don't have that kind of connection lol. They wouldn't listen to me either even if I did. They have no incentive to provide such a service.
